### PR TITLE
Add DigitalOcean to cloud provider documentation.

### PR DESCRIPTION
**Pull request by MorrisLaw**
Updated the cloud provider documentation to include DigitalOcean with a description and a reference to our `cloud-controller-manager`.

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/cloud-providers.md [hunk 1]
```diff
@@ -125,6 +125,15 @@ If you wish to use the external cloud provider, its repository is [apache/clouds
 The CloudStack cloud provider uses the hostname of the node (as determined by the kubelet or overridden with `--hostname-override`) as the name of the Kubernetes Node object.
 Note that the Kubernetes Node name must match the CloudStack VM name.
 
+## DigitalOcean
+
+If you wish to use the external cloud provider, its repository is [digitalocean/digitalocean-cloud-controller-manager](https://github.com/digitalocean/digitalocean-cloud-controller-manager)
+
+### Node Name
+
+The DigitalOcean cloud provider uses the hostname of the node (as determined by the kubelet or overridden with `--hostname-override`) as the name of the Kubernetes Node project.
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/cloud-providers.md:134]: One miswording
```suggestion
The DigitalOcean cloud provider uses the hostname of the node (as determined by the kubelet or overridden with `--hostname-override`) as the name of the Kubernetes Node object.
```

Comment (MorrisLaw) [reply content/en/docs/concepts/cluster-administration/cloud-providers.md]: Updated. Thank you!

**Comment by zacharysarah** [comment]
@MorrisLaw This PR includes [third party content](https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content) that falls outside accepted guidelines. We need to revert this PR. Sorry for any inconvenience.

**Comment by MorrisLaw** [comment]
> @MorrisLaw This PR includes [third party content](https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content) that falls outside accepted guidelines. We need to revert this PR. Sorry for any inconvenience.

I initially thought the violation was that the repo has to be within in Kubernetes (`kubernetes` and `kubernetes-sigs`) But, Tencent appears to have something outside of the `kubernetes` or `kubernetes-sigs` just like DigitalOcean does. https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#tencent-kubernetes-engine references https://github.com/TencentCloud/tencentcloud-cloud-controller-manager.

The one I wanted to add was https://github.com/digitalocean/digitalocean-cloud-controller-manager.

What exactly was the violation? _and_ What can I do to get a PR in that does not violate the community guidelines?

**Comment by MorrisLaw** [comment]
> @MorrisLaw This PR includes [third party content](https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content) that falls outside accepted guidelines. We need to revert this PR. Sorry for any inconvenience.

@zacharysarah I initially thought the violation was that the repo has to be within in Kubernetes (`kubernetes` and `kubernetes-sigs`) But, Tencent appears to have something outside of the `kubernetes` or `kubernetes-sigs` just like DigitalOcean does. https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#tencent-kubernetes-engine references https://github.com/TencentCloud/tencentcloud-cloud-controller-manager.

The one I wanted to add was https://github.com/digitalocean/digitalocean-cloud-controller-manager.

What exactly was the violation? _and_ What can I do to get a PR in that does not violate the community guidelines?

**Comment by zacharysarah** [comment]
@MorrisLaw 👋  I think we resolved your questions on Slack. Please let me know if you have any more questions!

**Pull request merged**